src/JSFile/addRmove.js

```diff
@@ -0,0 +1,87 @@
+export const saveTasks = (tasks) => {
+  // save tasks to localstorage
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+export const addTask = (description) => {
+  const tasks = JSON.parse(localStorage.getItem('tasks')) ?? [];
+  const newTask = {
+    id: Math.floor(Math.random() * 1000),
+    description,
+    completed: false,
+    index: tasks.length,
+  };
+  tasks.push(newTask);
+  return tasks;
+};
+
+export const removeTask = (id, tasks) => tasks.filter((task) => task.id !== id);
+
+export const editTask = (id, newDesc) => {
+  const tasks = JSON.parse(localStorage.getItem('tasks')) ?? [];
+  tasks.map((task) => {
+    if (task.id === id) {
+      task.description = newDesc;
+    }
+    return task;
+  });
+  return tasks;
+};
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirement, as I can see that you are using the `random` generated IDs for removing, and editing the description of the **Todo**. You should have to perform the operations but based on the index value, not based on the `id`. Kindly implement the things as per the project requirements, take a look at the below attached image of requirment to understand the issue.

### Requirement
![image](https://github.com/NalianAchei2020/To_Do_List/assets/66476329/db3c763a-ca3b-4009-adcd-d6ec8a8f963b)